NtupleTools/python/parameters/ztt.py

```diff
@@ -184,5 +220,24 @@
 
 
     # dicandidates of form: object1_object2_VarName = 'string expression for candidate'
-    'dicandidateVariables' : PSet(),
+    'dicandidateVariables' : PSet(
+        object1_object2_PZetaLess0p85PZetaVis = 'pZeta({object1_idx}, {object2_idx}) - 0.85*pZetaVis({object1_idx}, {object2_idx})',
+        object1_object2_pt_tt = 'PtDiTauSyst({object1_idx}, {object2_idx})',
+        object1_object2_MtTotal = 'MtTotal({object1_idx}, {object2_idx})',
+        object1_object2_MvaMet = 'getMVAMET({object1_idx},{object2_idx}).at(0)',
```

Comment:
Did you set this up so that if the runMVAMet option is not used, these branches will not be included in the tree?